packages/dialtone-vue2/components/button/button.test.js

```diff
@@ -278,6 +278,56 @@ describe('DtButton Tests', () => {
           });
         });
       });
+
+      describe('When button has unstyled set to true', () => {
+        it('Should have unstyled class and no d-btn classes except d-btn--unstyled', async () => {
+          await wrapper.setProps({
+            unstyled: true,
+          });
+
+          button = wrapper.find('.base-button__button');
+
+          // Should have the d-btn--unstyled class
+          expect(button.classes().includes('d-btn--unstyled')).toBe(true);
+
+          // Should NOT have the d-btn class
+          expect(button.classes().includes('d-btn')).toBe(false);
+
+          // Should NOT have any d-btn-- classes except d-btn--unstyled
+          const buttonClasses = button.classes();
+          const nonUnstyledBtnClasses = buttonClasses.filter(cls =>
+            cls.startsWith('d-btn--') && cls !== 'd-btn--unstyled',
+          );
+          expect(nonUnstyledBtnClasses.length).toBe(0);
+        });
```

Comment:
We should only have one assertion per test, these tests are doing too much.

Reply:
Updated. Give it a look.